internal/cmd/githubtemplatefuncs.go

```diff
@@ -37,21 +37,25 @@ type gitHubTagsState struct {
 }
 
 var (
-	gitHubKeysStateBucket          = []byte("gitHubLatestKeysState")
-	gitHubLatestReleaseStateBucket = []byte("gitHubLatestReleaseState")
-	gitHubReleasesStateBucket      = []byte("gitHubReleasesState")
-	gitHubTagsStateBucket          = []byte("gitHubTagsState")
+	gitHubKeysStateBucket     = []byte("gitHubLatestKeysState")
+	gitHubReleaseStateBucket  = []byte("gitHubReleaseState")
```

Comment:
Rather than combining the states for releases and latest releases in the same bucket and using an extra magic key (`<latest>`) to distinguish between them, please use a separate bucket for the new functions.

Reply:
Done. It now should use a `gitHubVersionReleaseStateBucket`